apps/pie-storybook/stories/testing/pie-radio.test.stories.ts

```diff
@@ -0,0 +1,191 @@
+        updateArgs({ checked: radioElement.checked });
+        console.info(JSON.stringify(event));
+    };
+
+    return html`
+        <pie-radio
+            ?checked="${checked}"
+            ?disabled="${disabled}"
+            ?defaultChecked="${defaultChecked}"
+            ?required="${required}"
+            name="${ifDefined(name)}"
+            .value="${value}"
+            status="${status}"
+            @change="${onChange}"
+            data-test-id="pie-radio">
+            ${sanitizeAndRenderHTML(slot)}
+        </pie-radio>`;
+};
+
+const ExampleFormTemplate: TemplateFunction<RadioProps> = ({
+    value,
+    name,
+    checked,
+    defaultChecked,
+    disabled,
+    required,
+    slot,
+}: RadioProps) => {
+    const [, updateArgs] = UseArgs();
+
+    const onChange = (event: InputEvent) => {
+        const radioElement = event.target as HTMLInputElement;
+        updateArgs({ checked: radioElement.checked });
+        console.info('change event fired');
+    };
+
+    return html`
+        <style>
+            pie-radio {
+                display: block;
+                margin-bottom: var(--dt-spacing-b);
+            }
+        </style>
+        <form id="testForm" @submit="${onSubmit}">
+            <pie-radio
+                .value="${value}"
+                name="${ifDefined(name)}"
+                ?checked="${checked}"
+                ?defaultChecked="${defaultChecked}"
+                ?disabled="${disabled}"
+                ?required="${required}"
+                @change="${onChange}"
+                data-test-id="pie-radio">
+                ${sanitizeAndRenderHTML(slot)}
+            </pie-radio>
+            <pie-button variant="secondary" type="reset">Reset</pie-button>
+            <pie-button type="submit">Submit</pie-button>
+        </form>
+        <div id="formDataOutput"></div>`;
+};
+
+export const Default = createStory<RadioProps>(Template, defaultArgs)();
+export const ExampleForm = createStory<RadioProps>(ExampleFormTemplate, defaultArgs)();
+
+const longLabel = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi pretium quam eget dolor imperdiet placerat. Aliquam sollicitudin erat sed est lobortis sollicitudin. Nam vulputate, mi vel finibus convallis, mi dolor molestie arcu, vel pulvinar urna neque et sapien. Aenean euismod faucibus turpis et efficitur. Sed porttitor dui at justo cursus pulvinar. Sed scelerisque aliquet diam sed feugiat. Fusce id lorem finibus, tempor nulla tempor, tincidunt odio. Mauris consequat lectus ex, eget lacinia dui finibus sit amet. Phasellus maximus posuere sapien eget condimentum. Nunc viverra pharetra blandit.';
+
+const radioPropsMatrix: Partial<Record<keyof RadioProps, unknown[]>> = {
+    checked: [true, false],
+    disabled: [true, false],
+    slot: ['Label', longLabel],
+    status: ['default', 'error'],
+};
+
+export const Variations = createVariantStory<RadioProps>(Template, radioPropsMatrix, { multiColumn: true });
```

Comment:
Can we add something like the below for the long longLabel for the screenshots? We're doing the implementation for labelling in pie-toast.

Can be addressed in a later PR if too much trouble now.

`const variantPropDisplayOptions: PropDisplayOptions<ToastProps> = {
    hiddenProps: ['duration'],
    propLabels: {
        message: {
            [longMessage]: 'With long message',
            [shortMessage]: 'With short message',
        },
    },
};`

Reply:
Fixed! Thanks for adding this! ❤️